rclpy/rclpy/utilities.py

```diff
@@ -18,6 +18,8 @@
 from rclpy.constants import S_TO_NS
 from rclpy.context import Context
 
+import ament_index_python
```

Comment:
flake8 is showing a failure, because this isn't imported in the correct order (the failure didn't appear in CI because `rclpy` was missing in the packages to be tested). @ahcorde Can you send a PR correcting it?

Reply:
https://github.com/ros2/rclpy/pull/519